setup.py

```diff
@@ -15,7 +15,7 @@
 SRC_DIR = os.path.join(THIS_DIR, "src")
 SNOWPARK_SRC_DIR = os.path.join(SRC_DIR, "snowflake", "snowpark")
 MODIN_DEPENDENCY_VERSION = "==0.30.1"  # Snowpark pandas requires modin 0.30.1, which is compatible with pandas 2.2.x
-CONNECTOR_DEPENDENCY_VERSION = ">=3.12.0, <4.0.0"
+CONNECTOR_DEPENDENCY_VERSION = ">=3.14.0, <4.0.0"
```

Comment:
it looks this we need new features in connector, have we made the change in sproc connector?

Reply:
yes. cc @sfc-gh-zli